canon/src/store/mod.rs

```diff
@@ -0,0 +1,91 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use cfg_if::cfg_if;
+
+use core::fmt;
+
+use crate::id::IdHash;
+use crate::CanonError;
+
+cfg_if! {
+    if #[cfg(feature = "host")] {
+        mod host;
+        use host::HostStore as Inner;
+    } else if #[cfg(feature = "bridge")] {
+        mod bridge;
+        use bridge::BridgeStore as Inner;
```

Comment:
Why renamed `hosted` in `bridge`?
I was never too fond of `host` and `hosted` but at least it was clear that they were mutually exclusive, with `host` and `bridge` we lost this kind of clarity.

Reply:
I think it's more clear, but i'm not passionate about it, we could change back.